app/helpers/textual_mixins/textual_region.rb

```diff
@@ -23,10 +23,17 @@ def textual_region_with_button_link
     url = @record.miq_region.remote_ui_url
     h = {:label => _('Remote Region'), :value => _('Connect to VM in its Region')}
     if url
-      h[:link] = url_for(:host => url, :action => 'show', :id => @record)
-      h[:title] = _('Connect to VM in its Region')
-      h[:external] = true
-      h[:button] = true
+      h = {
+        :label => _('Remote Region'), 
+        :button => {
+          :name => _('Connect to VM in its Region'),
+          :action => nil,
+          :href => url_for(:host => url, :action => 'show', :id => @record),
+          :target => "_blank",
+          :size => "small",
+          :disabled => false,
+        }
+      }
```

Comment:
Hey @DavidResende0 , could you please fix the lint errors.

just run the below command to fix them-

`rubocop app/helpers/textual_mixins/textual_region.rb -A`